vrp-pragmatic/src/format/solution/geo_serializer.rs

```diff
@@ -82,6 +88,18 @@ pub fn serialize_solution_as_geojson<W: Write>(
     serde_json::to_writer_pretty(writer, &geo_json).map_err(Error::from)
 }
 
+/// Serializes solution into pragmatic and geo json format.
+pub fn serialize_solution_with_geojson<W: Write>(
+    writer: &mut BufWriter<W>,
+    problem: &Problem,
+    solution: &Solution,
+) -> Result<(), Error> {
+    let geo = create_geojson_solution(problem, solution)?;
+    let combined = FeatureCollectionWithSolution{geo, pragmatic: solution.clone()};
```

Comment:
There must be a `space` between `{` and `geo`. Please run `cargo fmt` to get proper formatting here and everywhere.